tests/index_test.py

```diff
@@ -33,6 +33,13 @@ def test_multiindex_creation(self):
         with self.assertRaises(ValueError):
             idx = ak.MultiIndex([ak.arange(5), ak.arange(3)])
 
+    def test_is_unique(self):
+        i = ak.Index(ak.array([0, 1, 2]))
+        self.assertTrue(i.is_unique)
+
+        i = ak.Index(ak.array([0, 1, 1]))
+        self.assertTrue(i.is_unique is False)
```

Comment:
go back and forth on which is more readable, but i just wanted to throw out there that `assertFalse` is an option